notebooks/266-table-question-answering/266-table-question-answering.ipynb

```diff
@@ -0,0 +1,400 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    core = ov.Core()
</pre><p>you already have core initialized in the above cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>